streams/ci.m3u

```diff
@@ -9,7 +9,15 @@ https://nci-live.secure2.footprint.net/nci/nci.isml/.m3u8
 https://strhlslb01.streamakaci.tv/str_ntv_ntv/str_ntv_ntv_multi/playlist.m3u8
 #EXTINF:-1 tvg-id="RefletTV.ci",REFLET TV (1080p)
 https://edge-a3.evrideo.tv/8f37c9f0-fe22-44f4-b64a-76ad11730daf_1000026630_HLS/manifest.m3u8
-#EXTINF:-1 tvg-id="RTI1.ci",RTI 1
+#EXTINF:-1 tvg-id="RTI1.ci",RTI 1 (1080p)
 https://www.enovativecdn.com/rticdn/smil:rti1.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="RTI2.ci",RTI 2
+#EXTINF:-1 tvg-id="RTI2.ci",RTI 2 (1080p)
 https://www.enovativecdn.com/rticdn/smil:rti2.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="BenieTV.ci",Benie TV (720p)
+https://voozmedia.fun/benietv/livestream/playlist.m3u8
+#EXTINF:-1 tvg-id="ChristLive.ci",Christ Live (720p)
+https://voozmedia.fun/stream/christlive/playlist.m3u8
+#EXTINF:-1 tvg-id="MiracleTVPlus.ci",Miracle TV+ (720p)
+https://strhlslb01.streamakaci.tv/str_mtv_mtv/str_mtv_multi/playlist.m3u8
+#EXTINF:-1 tvg-id="RTILa3.ci",RTI La 3 (1080p) [Not 24/7]
+https://www.enovativecdn.com/rticdn/smil:rti3.smil/playlist.m3u8
```

Comment:
> https://www.enovativecdn.com/rticdn/smil:rti3.smil/playlist.m3u8

I have never saw this link working. Have you ever seen it online?

Reply:
Geo, for Ivoirian people only.